docs/hybrid.md

```diff
@@ -89,6 +89,8 @@ their `slurm.conf`, by default.
 
 ## Terraform
 
+> **NOTE:** Since it only supports v5, some links are only meant to be used on the v5 branch.
```

Comment:
you can just make a note at the top of this page saying hybrid clusters are only support on v5, please use v5 branch to access the required code and documentation.